scripts/size_report/report_binary_size.ts

```diff
@@ -77,7 +80,16 @@ async function generateReportForNPMPackages(): Promise<Report[]> {
   for (const info of packageInfo) {
     const packages = await findAllPackages(info.location);
     for (const pkg of packages) {
-      reports.push(...(await collectBinarySize(pkg)));
+      try {
+        reports.push(...(await collectBinarySize(pkg)));
+      } catch (e) {
+        // log errors and continue to the next package
+        console.log(
+          `failed to generate report for ${pkg}.
+           error: ${e}`
+        );
```

Comment:
Is it safe to swallow the error here?

Reply:
I added this because there are internal testing/demo projects which we fail to generate reports for because they are not built, which is WAI, and I don't want to add special handling for them. 

Also, we are not really swallowing them, the errors are logged, so I think we are fine.

Reply:
Sounds good. Was just trying to double check with you.